packages/presentation/lib/home_screen/home_widget.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:domain/use_cases/impl/palindrome_impl.dart';
+import 'package:flutter/material.dart';
+
+class HomeWidget extends StatefulWidget {
+  HomeWidget({Key? key}) : super(key: key);
+
+  @override
+  State<HomeWidget> createState() => _HomeWidgetState();
+}
+
+class _HomeWidgetState extends State<HomeWidget> {
+  final palindromeCaseImpl = PalindromeCaseImpl();
+  String isPalindrome = "";
```

Comment:
seems that a variable which starts from `is` should be boolean. so rename it please